build/common.sh

```diff
@@ -472,11 +488,8 @@ function kube::build::build_image() {
   kube::version::save_version_vars "${build_context_dir}/kube-version-defs"
 
   cp build/build-image/Dockerfile ${build_context_dir}/Dockerfile
-  if kube::build::is_osx; then
-    sed -i "" "s/KUBE_BUILD_IMAGE_CROSS/${KUBE_BUILD_IMAGE_CROSS}/" ${build_context_dir}/Dockerfile
```

Comment:
I don't see an easy way out here to coexist with OSX.

This claims to work fine.  I can't test:
sed -i'' 's/old/new/g' *

Using a global SED_ARGS="-i''" could also work or a kube::sed.

Sadly, none of these options allows you to work seamlessly without having to remember to do *something* different.